lib/features/map_view/widgets/bottom_scroll_sheet/bottom_scroll_sheet.dart

```diff
@@ -32,21 +33,24 @@ class BottomScrollSheet<T extends GoogleNavigable> extends ConsumerWidget {
             min(1.0, recomendedSheetHeight / screenHeight);
```

Comment:
[OFF TOP] Android studio complains that we have a typo here 😝 Should be _recommended_.

Reply:
damn. spelling is kinda hard :P corrected as suggested :)